source/tyr/tyr/tasks.py

```diff
@@ -144,6 +147,103 @@ def update_data():
         files = glob.glob(instance_config.source_directory + "/*")
         import_data(files, instance, backup_file=True)
 
+
+BANO_REGEXP = re.compile('.*bano.*\.csv')
+
+
+def type_of_autocomplete_data(filename):
+    """
+    return the type of autocomplete data of the files
+    filename can be either a directory, a file or a list of files
+
+    return can be:
+        - 'bano'
+        - 'osm'
+
+    """
+    def files_type(files):
+        #first we try fusio, because it can load fares too
+        if any(f for f in files if BANO_REGEXP.match(f)):
```

Comment:
done by changing the BANO regexp to *bano*, the .gz will be handled by bano2mimir